website/sidebars.json

```diff
@@ -61,7 +61,7 @@
         "type": "subcategory",
         "label": "Creating Contracts",
         "ids": [
-          "near-bindgen/near-bindgen",
+          "api/near-sdk-rs/index",
```

Comment:
@amgando note that this link doesn't show up in the sidebar: https://docs-pr-291.onrender.com/docs/roles/developer/contracts/assemblyscript

But the same exact `id` used on the `api` page, down on [line 161](https://github.com/nearprotocol/docs/pull/291/files#diff-cb1f944b76e14c5b994bf7231b98104aR161) below, _does_ generate the expected result in the sidebar: https://docs-pr-291.onrender.com/docs/api/near-sdk-rs/

Any idea how to fix this?

Reply:
Hmm, it looks like we might need to update to Docusaurus v2 to fix this? https://github.com/facebook/docusaurus/issues/868

How do you think we should proceed in the meantime?

Reply:
only the first `id` wins I think, or the last, can't recall. we don't need the page in the other location. let's just put all API docs in this section

I don't think we should rush docusaurus v2. after RL1 maybe? wdyt?